Source/BootloaderStorage.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef __BOOTLOADER_STORAGE_H__
+#define __BOOTLOADER_STORAGE_H__
+
+#include <inttypes.h>
+#include "eepromcontrol.h"
+
+extern char _BOOTLOADER;
+
+class BootloaderStorage {
+public:
+    bool erase(){
+        eeprom_unlock();
+        int ret = eeprom_erase_sector(FLASH_SECTOR_0);
+        ret |= eeprom_erase_sector(FLASH_SECTOR_1);
+#if !defined(OWL_MODULAR) && !defined(OWL_PEDAL)
+        ret |= eeprom_erase_sector(FLASH_SECTOR_2);
+        ret |= eeprom_erase_sector(FLASH_SECTOR_3);
+#endif
```

Comment:
Actually we don't have to pass that - firmware can just get it from linker variable. So unless there's a use case where we want to make partial erasure for bootloader, there no point passing that (since it would also be read from linker variable by the caller)